### PR TITLE
yaml manifest fix

### PR DESCRIPTION
**Pull request by hirenoble**
Fix for broken yaml manifest navigation
## Description
The homePage is not set and graphql query for homePage returns null, so navigating to site gets blank page.

When homePage / root url is undefined or '/' it will fetch the first available page in the nav as home page.

## Motivation and Context

Required to support yaml manifest file.
## How Has This Been Tested?
Tested locally in develop and build

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/fromYaml.js [hunk 1]
```diff
@@ -58,13 +82,19 @@ const fromYaml = (content) => {
     }
 
     const { order, title, name, url, pages } = object
+    const convertedPages = convertPages(pages)
 
+    let homePage = url
+    if (!homePage || homePage === undefined || homePage === '/') {
```
Comment (macdonst) [inline src/fromYaml.js:88]: Why `homePage === '/'`?

**Review by macdonst — CHANGES_REQUESTED** [review]
Code looks good but can you add a test where the homepage is not specified?

File: src/fromYaml.js (hunk 1 above)
Comment (hirenoble) [reply src/fromYaml.js:88]: Saw in this test, https://github.com/adobe/parliament-transformer-navigation/blob/main/src/__tests__/fromYaml.test.js#L21 that root url might be explicitly defined as '/'.

**Comment by hirenoble** [comment]
> Code looks good but can you add a test where the homepage is not specified?

@macdonst added test for the missing homepage, anything else pending here before we merge?

**Review by macdonst — APPROVED** [review]

**Pull request merged**